charts/tefca-viewer/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+---
+apiVersion: v2
+name: tefca-viewer-chart
+description: A Helm chart for Kubernetes
+type: application
+version: 0.1.8
```

Comment:
Since this is the first release, I think we could do 0.1.1? We haven't been matching versions across our charts. (Though maybe we should start...)